src/hera/v1/operator.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+
+class Operator(Enum):
+    """Operator is an logic representation.
+
+     This can be used on conditional tasks workflow.
+
+    Attributes
+     ----------
+     value: str
+         The value of an operator in str.
+    """
+
+    equals = '=='
+    greater = '>'
+    lower = '<'
+    greater_equal = '>='
+    lower_equal = '<='
+    not_equal = '!='
```

Comment:
Very picky suggestion:
```suggestion
class Operator(Enum):
    """Operator is a representation of mathematical comparison symbols.

     This can be used on tasks that execute conditionally based on the output of another task.

    Notes
    ------
    The task that outputs its result needs to do so using stdout. See `examples` for a sample workflow.
    """

    equals = '=='
    greater = '>'
    less = '<'
    greater_equal = '>='
    less_equal = '<='
    not_equal = '!='
```